plugins/bulk_upload/csv/batch/BulkUploadVendorCatalogItemEngineCsv.php

```diff
@@ -108,27 +118,7 @@ protected static function handlePriceColumns(&$pricing, $bulkUploadResult, $colu
 
 	protected function handleEnumColumns($value, $column, $bulkUploadResult)
 	{
-		switch($column)
-		{
-			case 'serviceFeature':
-				$enumValue = self::getEnumValue('KalturaVendorServiceFeature', $value);
-				break;
-
-			case 'serviceType':
-				$enumValue = self::getEnumValue('KalturaVendorServiceType', $value);
-				break;
-
-			case 'turnAroundTime':
-				$enumValue = self::getEnumValue('KalturaVendorServiceTurnAroundTime', $value);
-				break;
-
-			case 'outputFormat':
-				$enumValue = self::getEnumValue('KalturaVendorCatalogItemOutputFormat', $value);
-				break;
-
-			default:
-				$enumValue = null;
-		}
+		$enumValue = self::getEnumValue($this->columnToEnum[$column] ,$value);

```

Comment:
you have here an assomption that we will get a valid column name that will have enum value